assembl/static2/js/app/components/debate/navigation/timelineSegment.jsx

```diff
@@ -15,14 +12,9 @@ class TimelineSegment extends React.Component {
     const { isStepCompleted, isCurrentPhase, phaseIdentifier } = this.props;
     const { debateData } = this.props.debate;
     const slug = { slug: debateData.slug };
+
     if (isStepCompleted || isCurrentPhase) {
       browserHistory.push(`${get('debate', slug)}?phase=${phaseIdentifier}`);
-    } else {
-      const { locale } = this.props.i18n;
-      const startDate = getStartDatePhase(debateData.timeline, phaseIdentifier);
-      const phaseName = getPhaseName(debateData.timeline, phaseIdentifier, locale).toLowerCase();
-      const body = <div><Translate value="debate.notStarted" phaseName={phaseName} /><Localize value={startDate} dateFormat="date.format" /></div>;
-      displayModal(null, body, true, null, null, true);
     }
```

Comment:
when I read acceptance test, it's very clear. The PO expect to have the popin without redirection to `/debate`. I'm not sure that what you did will be approved by the UX and the PO. The screenshot shows an example of the popin in order to keep the same UI.